frontend/packages/console-app/src/components/volume-snapshot/create-volume-snapshot/create-volume-snapshot.tsx

```diff
@@ -229,7 +229,7 @@ const CreateSnapshotForm = withHandlePromise<SnapshotResourceProps>((props) => {
             <PVCDropdown
               namespace={namespace}
               onChange={handlePVCName}
-              id="claimName"
+              id="pvcDropdown"
```

Comment:
This should be changed to `data-test=...` per our [best practices](https://github.com/openshift/console/blob/master/frontend/packages/integration-tests-cypress/README.md#best-practices).  Also, pls read Cypress's [Selecting Elements: Best Practices](https://docs.cypress.io/guides/references/best-practices.html#Selecting-Elements) -thanks